mix.exs

```diff
@@ -9,14 +9,19 @@ defmodule Closex.Mixfile do
       config_path: "config/config.exs",
       deps_path: "deps",
       lockfile: "mix.lock",
-      elixirc_paths: elixirc_paths(Mix.env),
-      elixir: "~> 1.5",
+      elixirc_paths: elixirc_paths(Mix.env()),
+      elixir: "~> 1.6",
```

Comment:
does this prevent it from being installed with 1.5?

Reply:
I assume so. Bumped the version a minor version but maybe I should go major in case this is true?